torch/_meta_registrations.py

```diff
@@ -119,11 +119,7 @@ def meta_unsqueeze_(self, dim):
     # new size
     size = list(self.size())
     size.insert(dim, 1)
-    # return self.as_strided_(size, strides)
-    # This is incorrect and self.as_strided_(...) should be returned.
-    # However, as_strided_ doesn't have a meta impl and there's no straightforward way
-    # to inplace modify the strides from python.
-    return self.as_strided(size, strides)
+    return torch.ops.aten.as_strided_(self, size, strides)
```

Comment:
Why not just `self.as_strided_(size, strides)`?
Also, it should be `self.as_strided_(size, strides, self.storage_offset())`

Reply:
Sure if it works. Wasn't in the docs. Omitting storage offset should be harmless

Reply:
> Omitting storage offset should be harmless

Our doc says that it default to 0: https://pytorch.org/docs/stable/generated/torch.as_strided.html?highlight=as_strided#torch.as_strided
But it indeed defaults to self.storage_offset().

Opened an issue to fix that.

Reply:
@albanD could you also have the issue track docs for `as_strided_` if it is supported on Tensors?